ui/js/models/perf/alerts.js

```diff
@@ -217,6 +217,23 @@ treeherder.factory('PhAlerts', [
             return _.find(phAlertSummaryStatusMap, { id: this.status }).text;
         };
 
+        AlertSummary.prototype.notesChanged = (this.notes !== this.original_notes);
```

Comment:
`original_notes` => `originalNotes`

Reply:
Also, do you actually need to specify this here in the prototype? By default the property will be undefined which resolves to false, and it looks like you set the property as needed later.